dora/core/common/src/main/java/alluxio/metrics/MetricKey.java

```diff
@@ -2207,6 +2207,18 @@ public static String getSyncMetricName(long mountId) {
           .setMetricType(MetricType.COUNTER)
           .setIsClusterAggregated(true)
           .build();
+  public static final MetricKey CLIENT_BYTES_READ_ALLUXIO =
+      new Builder("Client.BytesReadAlluxio")
+          .setDescription("Total number of bytes read from Alluxio by this client")
```

Comment:
```suggestion
          .setDescription("Total number of bytes read from Alluxio by this client. "
              + "This includes both the case where the data has been cached on the "
              + "workers, and the case where the workers need to load it from the "
              + "UFS.")
```

Reply:
How about read from client local cache? does it also count as "read from Alluxio"?

Reply:
I guess apart from "reading from UFS directly," reading from any kind of cache offered by Alluxio counts as "reading from Alluxio"? The cache would include worker cache as well as client local cache.

Reply:
Then maybe name it `BytesReadFromWorker`?

Reply:
OK. I will rename the `BytesWrittenAlluxio` too.

Reply:
renamed all metrics to better names.